### ISSUE TITLE
chat not working on iOS 

### ISSUE THREAD

**Issue by boy13epic** [hrydgard/ppsspp#13065]
### What happens?
no type here 

### What *should* happen?
a type hère in the chat

### What hardware, operating system, and PPSSPP version?  On desktop, GPU matters for graphical issues.
iPad 6 JIT,iOS,ppsspp 1.10.0

**Comment by boy13epic** [comment]
![6F82C1E1-6A62-4935-BF81-4C9A2994FF8C](https://user-images.githubusercontent.com/62749891/85941919-470ab000-b926-11ea-9d0b-2e5359cc504b.jpeg)
this is what happens

**Comment by unknownbrackets** [comment]
There's probably no UI setup for a keyboard on that platform.

-[Unknown]

**Comment by adenovan** [comment]
yep on the last commit merged by ppsspp its just windows and android gui , no sdl or qt ui

**Comment by vnctdj** [comment]
You're wrong @adenovan, the chat UI is available on SDL and Qt too.

**Comment by adenovan** [comment]
so there is some || in latest master is QT working fine sdl should not be a problem?

there is no platform ios on elif its we re missing button like on android
